pkg/verify/sct.go

```diff
@@ -29,16 +30,18 @@ import (
 // leaf certificate, will extract SCTs from the leaf certificate and verify the
 // timestamps using the TrustedMaterial's FulcioCertificateAuthorities() and
 // CTLogs()
-func VerifySignedCertificateTimestamp(leafCert *x509.Certificate, threshold int, trustedMaterial root.TrustedMaterial) error { // nolint: revive
+func VerifySignedCertificateTimestamp(chains [][]*x509.Certificate, threshold int, trustedMaterial root.TrustedMaterial) error { // nolint: revive
+	if len(chains) == 0 && len(chains[0]) == 0 && chains[0][0] == nil {
+		return errors.New("no chains provided")
+	}
 	ctlogs := trustedMaterial.CTLogs()
-	fulcioCerts := trustedMaterial.FulcioCertificateAuthorities()
 
-	scts, err := x509util.ParseSCTsFromCertificate(leafCert.Raw)
+	scts, err := x509util.ParseSCTsFromCertificate(chains[0][0].Raw)
```

Comment:
Can we add a comment noting that chains[<any>][0] is always the leaf certificate? This just stands out as looking problematic when it should be safe.